lib/python/pyflyby/_interactive.py

```diff
@@ -1881,61 +1763,90 @@ def _enable_completer_hooks(self, completer):
         # manages, is not useful because that only works for specific commands.
         # (A "command" refers to the first word on a line, such as "cd".)
         #
-        # We choose to advise global_matches() and attr_matches(), which are
-        # called to enumerate global and non-global attribute symbols
-        # respectively.  (python_matches() calls these two.  We advise
-        # global_matches() and attr_matches() instead of python_matches()
-        # because a few other functions call global_matches/attr_matches
-        # directly.)
+        # We avoid advising attr_matches() and minimise inference with
+        # global_matches(), because these are not public API hooks,
```

Comment:
```suggestion
        # We avoid advising attr_matcher() and minimise inference with
        # global_matcher(), because these are not public API hooks,
```